src/astra_driver.cpp

```diff
@@ -729,6 +729,15 @@ sensor_msgs::msg::CameraInfo::SharedPtr AstraDriver::getDepthCameraInfo(int widt
 void AstraDriver::readConfigFromParameterServer()
 {
   depth_frame_id_ = std::string("openni_depth_optical_frame");
+
+  // Load the depth registration parameter, which may have been set before
+  //   driver initialization.
+  pnh_->get_parameter("depth_registration", depth_registration_);
+  if(depth_registration_)
+  {
+    ROS_INFO("Astra depth registration enabled");
+  }
```

Comment:
I think it's not required to print both cases. Maybe it would make sense in a DEBUG log but I don't think that the default behavior should be to print the state of every parameter enable/disable. The driver will be way too verbose in the long run if we take that approach